### ISSUE TITLE
Latest update (0.7.4) broke the multi module kmp build 

### ISSUE THREAD

**Issue by sureshg** [Kotlin/kotlinx-kover#488]
**Describe the bug**
After upgrading to `0.7.4` from `0.7.3`, gradle started throwing an   `implicit_dependency` dependency error for `koverVerify` tasks.

I do have a multi module kmp project setup, where in apply kover plugin to all sub-projects using a pre-compiled script plugin

```
:benchmark: benchmark
:client: client
:web: web
:backend:jvm: jvm
:meta:compiler: compiler
:meta:ksp: ksp
:meta:compiler:plugin: plugin
:meta:ksp:processor: processor
```
then in the root build script (for merged report).

```
 dependencies {
    allprojects.forEach {
       kover(it)
    }
  }
```

The client and meta:compiler projects are independent and doesn't have any dependencies except kover plugin is applied to both.

**Errors**
```
Some problems were found with the configuration of task ':client:koverVerify' (type 'KoverVerifyTask').
  - Gradle detected a problem with the following location: '/../meta/compiler/plugin/build/kover/bin-reports/test.ic'.

    Reason: Task ':client:koverVerify' uses this output of task ':meta:compiler:plugin:test' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.

    Possible solutions:
      1. Declare task ':meta:compiler:plugin:test' as an input of ':client:koverVerify'.
      2. Declare an explicit dependency on ':meta:compiler:plugin:test' from ':client:koverVerify' using Task#dependsOn.
      3. Declare an explicit dependency on ':meta:compiler:plugin:test' from ':client:koverVerify' using Task#mustRunAfter.

    For more information, please refer to https://docs.gradle.org/8.4/userguide/validation_problems.html#implicit_dependency in the Gradle documentation.
  - Gradle detected a problem with the following location: '.../../meta/compiler/plugin/build/generated/ksp/main/kotlin'.

    Reason: Task ':client:koverVerify' uses this output of task ':meta:compiler:plugin:kspKotlin' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.

```

**Environment**
 - Kover Gradle Plugin version: 0.7.4
 - Gradle version: 8.4
 - Kotlin project type: Kotlin/Multiplatform=
 - Other context important for this bug: MacOS (arm64): 13.6 Ventura

**Comment by shanshin** [comment]
It looks like this problem is about relative paths. See the [issue](#478) and [comment](https://github.com/Kotlin/kotlinx-kover/issues/478#issuecomment-1758672214)

**Comment by shanshin** [comment]
Closed in favor of #478
